portfolio/src/main/webapp/scripts/animations.js

```diff
@@ -1,29 +1,50 @@
-/* intro section animations */
-
-gsap.to('.layer-1', { y: '-100vh', delay: .5 });
-gsap.to('.layer-2', { y: '-100vh', delay: .7 });
-gsap.to('.layer-3', { y: '-100vh', delay: .9 });
-gsap.to('#overlay', { y: '-100vh', delay: 1.5 });
-gsap.to('.circle', 20, { rotation:"360", ease: Linear.easeNone, repeat: -1 });
-
-setTimeout(() =>
-  type('big-header-main', '>Hi, my name is <span style="color: #9366e3; font-weight: bold;">Sabrina</span>', 1),
-  2000
-)
-setTimeout(() =>
-  type('sub-header-main', '>I like to code', 1),
-  5000
-)
+// Color constants
+const OFF_WHITE = '#ede7f6';
+const PORTFOLIO_HOVER_BLUE = '#00bcd4';
+const SKILL_DOT_COLOR = '#b39ddb';
+const SKILL_DOT_FLIPPED_COLOR = '#673ab7';
 
-gsap.fromTo('#portfolio-button', 2,
-  { opacity: 0 }, 
-  { x: '0', opacity: 1, ease:'back.out(1.7)', delay: 7.5 }
-);
+// --Intro section animations--
+
+/* Overlay */
+gsap.to('.layer-left', {y: '-100vh', delay: .5});
+gsap.to('.layer-middle', {y: '-100vh', delay: .7});
+gsap.to('.layer-right', {y: '-100vh', delay: .9});
+gsap.to('#overlay', {y: '-100vh', delay: 1.5});
+
+/* Rotate circles */
+gsap.to('.circle', 20, {rotation:"360", ease: Linear.easeNone, repeat: -1});
 
-/* Typewriter effect for title*/
-function type(id, text, i = 0, isTag = false) {
-  if (i === text.length) return;
-  var newText = text.slice(0, ++i); // get text + new char
+/* Fade in portfolio button */
+gsap.fromTo('#portfolio-button', 2,
+    {opacity: 0}, 
+    {x: '0', opacity: 1, ease:'back.out(1.7)', delay: 5.5});
+
+// --Typewriter effect for title--
+const mainHeader = document.getElementById('big-header-main')
+const name = document.getElementById('name')
+const subHeader = document.getElementById('sub-header-main')
+
+ityped.init(mainHeader, { 
+  strings: ['Hi, my name is Sabrina'], 
+  typeSpeed: 70, 
+  startDelay: 2000, 
+  loop: false, 
+  showCursor: false,
+});
+
+ityped.init(subHeader, { 
+  strings: ['I like to code'], 
+  typeSpeed: 70, 
+  startDelay: 4000, 
+  loop: false, 
+  showCursor: false,
+});
+
+/* Appends the given text character-by-character for a typewriter effect */
+function type(id, text, startIndex = 0, isTag = false) {
```

Comment:
Looks like you decided to import a library to do this instead. Should we delete this?

Reply:
Whoops, forgot to do that. Thanks 👍